packages/frontend/src/components/navbar/Banner.tsx

```diff
@@ -3,12 +3,12 @@ import React from 'react'
 export function Banner() {
   return (
     <a
-      className="block text-center bg-blue-500 text-black text-xs font-bold px-4 py-1.5"
-      href="https://grant-explorer.gitcoin.co/#/round/1/0xe575282b376e3c9886779a841a2510f1dd8c2ce4/0x4a8d8bb79a51a2b0e3d136c15b672ea87c9fd7dc5b0c3cd76674d5ebe4f7dac0-0xe575282b376e3c9886779a841a2510f1dd8c2ce4"
+      className="block text-center bg-pink-200 dark:bg-purple-700 text-xs font-medium px-4 py-1.5"
```

Comment:
Can we use the same color scheme as the Jobs `Hiring` badge?

Reply:
Yeah, I tried it before with `font-bold` and didn't like it so that's why I changed the colors. Actually, with font-medium, it looks good. Fixed

Reply:
Yeah, I tried it before with font-bold and didn't like it so that's why I changed the colors. Actually, with font-medium, it looks good. Fixed